releasenotes/releasenotes_develop.md

```diff
@@ -1,52 +1,343 @@
-# Release Notes develop
+# Release Notes 0.8.0
 
-...
+Keptn 0.8 improves the core use cases of continuous delivery and automated operations by implementing the new Shipyard version [v0.2](https://github.com/keptn/spec/tree/0.2.0). This new Shipard version has been proposed and refined in [KEP 06](https://github.com/keptn/enhancement-proposals/pull/6).
 
-**The key announcements of Keptn x.x.x**:
+---
 
+**Key announcements:**
+
+:rocket: *Separated and explicit Shipyard-defined processes (aka. task sequences) for continuous delivery / automated remediation*: With this Keptn release, it is possible to have multiple processes (called *task sequences*) within one stage. These task sequences are separated from each other and list the tasks that are triggered sequentially.
+
+:tada: *Support individual tasks in sequences*: It is now possible to add custom tasks to a *task sequence* to address needs of delivery/remediation use-cases that go beyond the opinionated approach Keptn is offering.
+
+:star: *Trigger of a sequence can be configured - allowing multiple parallel stages*: The new Shipyard supports the definition of triggers that launch the execution of a *task sequences*. This helps to make it explicit when a sequence gets triggers. Besides, this linking mechanism allows connecting multiple sequences (of different stages) to listen to the same trigger. Consequently, it is possible to connect multiple stages, which are on the same level, to one preceding stage - as shown below:
+
+> *Screenshot here*
+
+:star2: *New types of events*: In course of implement the new Shipyard version in Keptn, the Keptn Cloud-events were streamlined and follow now a common pattern. Basically, Keptn just sends out an event of type: `sh.keptn.event.{task.name}.triggered` and other services react on: 
+  * `sh.keptn.event.{task.name}.triggered`      > *sent out by Keptn*
+  * `sh.keptn.event.{task.name}.started`        > *sent out by a Keptn-service when starting the tasks*
+  * `sh.keptn.event.{task.name}.status.changed` > *sent out by a Keptn-service to inform about a status update*
+  * `sh.keptn.event.{task.name}.finished`       > *sent out by a Keptn-service when the task is finished*
+
+:sparkles: *Multi-cluster support*: Based on the implementation of Shipyard v0.2, Keptn - as a control-plane for delivery and remediation - is now capable of serving multiple clusters. This is known as the split between *Control plane* and *Execution plane*. For this use-case, the Keptn project offers to run the helm-service (to deploy) and jmeter-service (to test) on the *Execution plane*. This *Execution plane* can be on a cluster other than the cluster where Keptn is installed. 
+
+:dizzy: *Sequence screen in Keptn Bridge*: The new capabilities of Keptn for dealing with task sequences received a dedicated screen in the Keptn Bridge. This screen provides filtering capabilities and a stage-divided view on the performed delivery or remediation tasks. 
+
+> *Screenshot here*
```

Comment:
Don't forget to add the screenshot for the final version

Reply:
Thanks for the reminder. I will add it on the GitHub Release page then.